src/browser/modules/Stream/CypherFrame/helpers.ts

```diff
@@ -443,9 +444,13 @@ export function mapNeo4jValuesToPlainValues(values: any): any {
  */
 function neo4jValueToPlainValue(value: any) {
   switch (get(value, 'constructor')) {
+    case neo4j.types.Duration:
```

Comment:
Can we move this code that converts the value to a duration string to a shared util function or something?

Reply:
Done!